pkg/configtx/consortiums.go

```diff
@@ -29,25 +29,17 @@ func (c *ConfigTx) RemoveConsortium(consortiumName string) {
 	delete(consortiumGroup, consortiumName)
 }
 
-// AddOrgToConsortium adds an org definition to a named consortium in a given
+// SetConsortiumOrg adds an org definition to a named consortium in a given
 // channel configuration.
-func (c *ConfigTx) AddOrgToConsortium(org Organization, consortium string) error {
+// If an Orgnization already exist in current configuration, its value will be overwritten.
```

Comment:
`If the consortium org already exists in the current configuration, its value will be overwritten.`

Reply:
done